crates/ruff_dev/src/check_formatter_stability.rs

```diff
@@ -54,6 +57,33 @@ struct WrapperArgs {
 }
 
 pub(crate) fn main(args: &Args) -> anyhow::Result<ExitCode> {
+    let all_success = if args.multi_project {
+        let mut all_success = true;
+        for dir in args.files[0].read_dir()? {
```

Comment:
Why only the first path?

Reply:
that should have been only my dummy, fixed